libtiny_tui/src/msg_area/line.rs

```diff
@@ -85,35 +80,48 @@ impl Seg {
     }
 }
 
+impl Deref for StyledString {
+    type Target = String;
+    fn deref(&self) -> &Self::Target {
+        &self.inner
+    }
+}
+
+impl DerefMut for StyledString {
+    fn deref_mut(&mut self) -> &mut Self::Target {
+        &mut self.inner
+    }
+}
```

Comment:
This is considered an anti-pattern by the community: https://github.com/rust-unofficial/patterns/blob/master/anti_patterns/deref.md we also consider this an anti-pattern in the code base at work. I don't think not having this will cause a lot of pain, doing `self.inner.foo` instead of `self.foo` and it's easier on the reader, so let's remove these.

Reply:
Thanks for the insight. Will remove.